bin/util.js

```diff
@@ -74,6 +76,29 @@ module.exports = {
             return _.split(list, ',');
```

Comment:
We just need to do this:
```suggestion
            return list.split(',').map((item) => { return item.trim(); });
```

Reply:
@codenirvana I tried this, but I believe commander for variadic parameters will run this function for each parameter individually i.e. if I give ```-r cli, json```, in the above function ```list``` would first be ```"cli"``` and then ```" json"```, therefore the final reporters parsed would just be ```['json']``` coming from the last ```list``` value.

While if we do not choose variadic parameters, commander will stop parsing after the space between ```cli, json``` and we won't be able to achieve this functionality.

Reply:
I have reduced the function to this now,
```js
let reporters = [];

/**
         * Helper to parse variadic parameters for reporters to an array.
         *
         * eg. item1,item2
         *
         * @param {String} list - The comma separated parameters.
         * @returns {String[]} - [item1, item2] - The array representation of the passed parameters.
         */
        reporterParse: (list) => {
            list.split(',').forEach((name) => {
                if (name.length) {
                    reporters.push(name);
                }
            });

            return reporters;
        },

```

It's a little bit more concise than before and works the same. Should I commit this change?

Reply:
What I meant was, the only change required is to trim the `csvParse` values, no commander change required.

Because variadic option will allow input like: `-r cli json` which is a major change in my opinion.
We can introduce this behavior change in v6 for all similar options and deprecate this.

Reply:
@codenirvana Sorry for the misunderstanding there.

But if I do not use variadic parameters with reporters, commander stops parsing the parameters after the space i.e. if I use ```-r cli, json``` the ```list``` argument in ```csvParse``` will only have the value ```cli,``` nothing else.